runtime/codert_vm/jswalk.c

```diff
@@ -186,7 +186,7 @@ UDATA  jitWalkStackFrames(J9StackWalkState *walkState)
 		walkState->outgoingArgCount = walkState->argCount;
 
 		if ((!(walkState->flags & J9_STACKWALK_SKIP_INLINES)) && getJitInlinedCallInfo(walkState->jitInfo)) {
-			jitGetMapsFromPC(walkState->currentThread, walkState->jitInfo, (UDATA) walkState->pc, &(walkState->stackMap), &(walkState->inlineMap));
+			jitGetMapsFromPC(walkState->walkThread, walkState->jitInfo, (UDATA) walkState->pc, &(walkState->stackMap), &(walkState->inlineMap));
```

Comment:
This modification causes functional change in `findOrCreateMapTable()`. Bit `J9_PRIVATE_FLAGS2_ASYNC_GET_CALL_TRACE` is going to be checked for different thread. Is it expected?
https://github.com/eclipse-openj9/openj9/blob/ed430d715a4bc83bff3eac3e214a5d719a2bc7c1/runtime/compiler/runtime/MethodMetaData.c#L234

Reply:
This is true, though in practice doesn't matter today, as ASGCT will always be calling with the walkThread being the current thread. It's unlikely but possible this restriction will be lifted in the future.

See my comment below - this is a perfect example of the confusion that will arise by doing this.